README.md

```diff
@@ -50,6 +52,7 @@ Then to install buildah follow the steps in this example:
   git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah 
   cd ./src/github.com/projectatomic/buildah 
   make 
+  make install
   ./buildah --help
```

Comment:
After _make install_, do we still need to run the tool from the current directory?  Not a blocker either way.